intra_process_comms/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>intra_process_comms_example</name>
+  <version>0.0.0</version>
+  <description>Prototype of intra-process communications.</description>
+  <maintainer email="william@osrfoundation.org">William Woodall</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <buildtool_depend>rosidl_default_generators</buildtool_depend>
```

Comment:
The package does not contain any messages and therefore neither requires `rosidl_default_generators` nor `rosidl_default_runtime`.